src/resolvers-cassandra/Edges/queries.js

```diff
@@ -8,71 +8,6 @@ const { makeSet, makeMap, makeMultiMap } = require('../../utils/collections');
 const { trackEvent } = require('../../clients/appinsights/AppInsightsClient');
 
 /**
- * @param {{site: string, fromDate: string, toDate: string, sourceFilter: string[]}} args
- * @returns {Promise.<{runTime: string, edges: Array<{name: string}>}>}
- */
-function terms(args, res) { // eslint-disable-line no-unused-vars
-  return new Promise((resolve, reject) => {
-    const query = `
-    SELECT topic
-    FROM fortis.watchlist
-    `.trim();
-
-    const params = [
-    ];
-
-    cassandraConnector.executeQuery(query, params)
-    .then(rows => {
-      const topics = makeSet(rows, row => row.topic);
-      const edges = Array.from(topics).map(topic => ({name: topic}));
-
-      resolve({
-        edges
-      });
-    })
-    .catch(reject);
-  });
-}
-
-/**
- * @param {{site: string}} args
- * @returns {Promise.<{runTime: string, edges: Array<{name: string, coordinates: number[]}>}>}
- */
-function locations(args, res) { // eslint-disable-line no-unused-vars
```

Comment:
We may need to keep this method around since
(1) it's still referenced in Edges/index.js on line 6 above
(2) it's used by the frontend in services.js/fetchEdges which is used by load_localities